### PR TITLE
Adding missing package information

### PR DESCRIPTION
**Pull request by acabarbaye**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,3 @@
 graft mbed_tools_ci_scripts
 global-exclude *.py[cod] __pycache__ *.so
+global-include *.spdx
```
Comment (mark-edgeworth) [inline MANIFEST.in]: Nit: missing newline at EOF

Comment (muchzill4) [inline MANIFEST.in]: Imma re-provision this with cookiecutter, so this is not necessary.

File: mbed_tools_ci_scripts/utils/aws_helpers.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+"""Helpers for interacting with AWS services such as S3.
```
Comment (muchzill4) [inline mbed_tools_ci_scripts/utils/aws_helpers.py:1]: This is not used by any common CI stuff, just by `mbed-targets`. I'd rather not have it in here until this is used by more than one package.

File: setup.py [hunk 3]
```diff
@@ -12,10 +12,26 @@
 with open(os.path.join(repository_dir, SOURCE_DIR, '_version.py')) as fh:
     exec(fh.read())
 
+# Use readme needed as long description in PyPI
+with open(os.path.join(repository_dir, 'README.md')) as fh:
```
Comment (muchzill4) [inline setup.py]: Similar comment to the first one, this will likely be blatted / re-done with cookiecuttering over this repo.

File: setup.py [hunk 4]
```diff
@@ -33,5 +49,14 @@
         'toml',
         'python-dotenv',
         'twine',
+        'boto3',
     ],
+    license='Apache 2.0',
+    long_description_content_type='text/markdown',
+    long_description=long_description,
+    name=PACKAGE_NAME,
+    packages=find_packages(),
```
Comment (muchzill4) [inline setup.py]: Any reason for using `find_packages()` vs `packages=['mbed_tools_ci_scripts']`? 🤔

Comment (acabarbaye) [reply setup.py]: this is how you set it up

Comment (muchzill4) [reply setup.py]: D'oh. Let me answer my own question then: "No reason and you should be ashamed of yourself for lack of consistency".

File: mbed_tools_ci_scripts/utils/aws_helpers.py (hunk 2 above)
Comment (acabarbaye) [reply mbed_tools_ci_scripts/utils/aws_helpers.py:1]: Well it is a CI utility and I rather have them all in one place. Plus, it is not really mbed-targets specific

File: MANIFEST.in (hunk 1 above)
Comment (acabarbaye) [reply MANIFEST.in]: How do you want me to proceed?
Do you want to create a separate PR where we sync using cookie cutter?

File: setup.py (hunk 4 above)
Comment (acabarbaye) [reply setup.py]: these were not my words ;)

**Review by mark-edgeworth — CHANGES_REQUESTED** [review]

File: mbed_tools_ci_scripts/utils/aws_helpers.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+
+def _get_aws_config() -> Tuple[str, dict]:
+    s3_region = configuration.get_value('AWS_DEFAULT_REGION')
+    s3_config = {
+        "aws_access_key_id": configuration.get_value('AWS_ACCESS_KEY_ID'),
+        "aws_secret_access_key": configuration.get_value(
+            'AWS_SECRET_ACCESS_KEY'),
+    }
+    return s3_region, s3_config
```
Comment (mark-edgeworth) [inline mbed_tools_ci_scripts/utils/aws_helpers.py:25]: Can't decide if this really needs to be in a separate function.

File: setup.py (hunk 4 above)
Comment (mark-edgeworth) [reply setup.py]: lovin' the self-punishment there...

**Review by mark-edgeworth — APPROVED** [review]

File: mbed_tools_ci_scripts/utils/aws_helpers.py (hunk 5 above)
Comment (acabarbaye) [reply mbed_tools_ci_scripts/utils/aws_helpers.py:25]: I followed the style of what we do elsewhere to get the config (i.e. generate docs)

**Pull request merged**